tests/apps/actorload/makefile

```diff
@@ -0,0 +1,19 @@
+export GO111MODULE ?= on
```

Comment:
nit: name file `Makefile`, with capital M.

Reply:
I cannot rename to Makefile :(

Reply:
ah I changed.